python/requirements/development.txt

```diff
@@ -19,3 +19,4 @@ pysam
 PyVCF
 python_jsonschema_objects
 biopython
+Bio
```

Comment:
This isn't BioPython, it's a different package; change to 'biopython'

Reply:
Really? My installation claimed it was missing the "Bio" module when testing. Doing `pip install Bio` seemed to fix it. Is it used somewhere else, perhaps?

Reply:
biopython is already in development.txt. So not sure why this didn't work before for me. But I guess this PR can simply be cancelled